src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -920,10 +920,13 @@ impl<T: 'static> EventProcessor<T> {
                                 },
                             });
 
-                            // Issue key press events for all pressed keys
-                            self.handle_pressed_keys(
+                            // Issue key press events for all pressed keys.
```

Comment:
Based on the other comments in this diff, seems like there shouldn't be punctuation at the end?

Reply:
LOL, I managed to completely skip reading that. I was even wondering why it was showing up in the diff.